src/main/java/com/lucas/plugintest/PingBlockCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.lucas.plugintest;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class PingBlockCommand implements CommandExecutor {
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String s, String[] args) {
+        
+        if (!(sender instanceof Player)) {
+            sender.sendMessage("Esse comando só pode ser executado por players!");
+            return false;
+        }
+
+        Player player = (Player) sender;
+
+        if (args.length == 0) {
+            int playerPing = player.getPing();
+            player.sendMessage("Seu ping é de : " + ChatColor.GREEN + playerPing + "ms");
```

Comment:
Maybe in the future it will be a good idea to print this with a color based on the ping, like the from 0 to 100 - green, 100 to 250 - yellow, > 250 red.